test/jdk/java/net/httpclient/CancelRequestTest.java

```diff
@@ -379,7 +379,7 @@ public void testGetSendAsync(String uri, boolean sameClient, boolean mayInterrup
             assertEquals(cf2.isCancelled(), false);
             assertEquals(latch.getCount(), 0);
 
-            var error = TRACKER.check(1,
+            var error = TRACKER.check(100,
```

Comment:
Hello Daniel, looking at the implementation in the `check` method of `ReferenceTracker`, it has this:

```
public AssertionError check(long graceDelayMs,
                            Predicate<Tracker> hasOutstanding,
                            String description,
                            boolean printThreads) {
    AssertionError fail = null;
    graceDelayMs = Math.max(graceDelayMs, 100);
...
```
So, even when we were passing it `1` milli seconds as a grace delay, it was changing it to `100` milli seconds. So I suspect the change in this PR won't have the intended effect since it would still wait the same amount of time and would potentially continue to have intermittent failures.

Reply:
Ah! Good catch.

Reply:
I have uploaded a new version - using 200ms instead on 100ms, and simply looking at the specific client that has been used for the operation (instead of looking at *all* of them).